packages/typescript-estree/src/ts-estree/ts-nodes.ts

```diff
@@ -11,11 +11,67 @@ declare module 'typescript' {
 
 export type TSToken = ts.Token<ts.SyntaxKind>;
 
+export type TSNodeSkipped =
```

Comment:
Could you please add a comment about why a node might be listed in this list?

Reply:
this was just created as middle step, i'm going to delete this